packages/Breadcrumbs/Breadcrumbs.md

```diff
@@ -1,3 +1,13 @@
+### Usage criteria
+
+- Breadcrumb links must link out to live pages on T.com
+- Use plain language to describe the page title
+- The recommendation is up to 5 links in the breadcrumb path
+- All links in the breadcrumb are clickable
+- The breadcrumb appears on the 3rd tier of the navigation
+- In desktop we recommend the breadcrumb list to 1 line. If the breadcrumb needs to wrap, we recommend a maximum of 2 lines
```

Comment:
> - On desktop viewports, the `Breadcrumbs` should be displayed inline, without wrapping. If the `Breadcrumbs` need to wrap ...

Please also change any self-reference to Breadcrumbs to use code blocks (wrap with backticks)